### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Combinatorics/SimpleGraph/Connectivity/Subgraph.lean
Comment (ericrbg) [inline Mathlib/Combinatorics/SimpleGraph/Connectivity/Subgraph.lean:151]: I kind of preferred the old one, but I think often I don't stick to mathlib style, so i'll let someone else decide

**Review by ericrbg — APPROVED** [review]
as usual someone else should check through it, but lgtm! thanks so much for tidying :)

maintainer merge

File: Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean
Comment (riccardobrasca) [inline Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean:184]: I see that we use both `tfae` and `TFAE`... do you know if there is an "official" decision about this?

File: Mathlib/CategoryTheory/Abelian/Pseudoelements.lean
Comment (riccardobrasca) [inline Mathlib/CategoryTheory/Abelian/Pseudoelements.lean:463]: There is no way to make this clickable in the doc, right?

**Review by riccardobrasca — APPROVED** [review]
Just two very small comments, otherwise LGTM, thanks!

bors d+

File: Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean
Comment (Ruben-VandeVelde) [reply Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean:184]: Zulip seems to lean towards `tfae` at the moment, so I'll leave it for now

**Comment by Ruben-VandeVelde** [comment]
bors merge

File: Mathlib/CategoryTheory/Abelian/Pseudoelements.lean
Comment (Ruben-VandeVelde) [reply Mathlib/CategoryTheory/Abelian/Pseudoelements.lean:463]: This works now: https://github.com/leanprover/doc-gen4/pull/151

**Pull request closed without merging**
